CONTRIBUTING.md

```diff
@@ -270,9 +270,9 @@ For other integer types without a format, cast `signed` types to `intmax_t` and
 Logging Conventions
 -----------------------------------
 
-CFEngine outputs messages about what its doing using the `Log()` function. It
-takes a `LogLevel` enum mapping closely to syslog priorities. Please try to do
-the following when writing output messages.
+CFEngine outputs messages about what it's doing using the `Log()` function. It
+takes a `LogLevel` enum mapping closely related to syslog priorities. Please try
```

Comment:
~related~

I prefer how it was before, but you can maybe add a comma after `enum`.

Reply:
The comma helps me. Otherwise I was reading it as "enum mapping" instead of an "enum (which is mapping) ..." :+1: